commands/eureka-run.js

```diff
@@ -20,17 +21,20 @@ program
     command = cmd.join(' ')
   })
   .option('-t, --tier [tier]', 'Run on machine of type [tier]', 'n1-standard-1')
+  .option('-n, --name [name]', 'Assign a name to the task', names.getRandomName().replace(/_/g, '-'))
 
   .parse(process.argv)
 
 if (!machineName || !command) {
   program.help()
+} else if (!program.name.match(/^[a-z0-9][-a-z0-9]{0,50}[a-z0-9]$/)) {
+  console.error('Invalid task name, only lowercase alphanumeric characters allowed')
```

Comment:
Please use printError for consistency (or at least console log with color instead of console.error)